apps/ledger-live-mobile/src/components/WebPlatformPlayer/index.js

```diff
@@ -542,6 +547,40 @@ const WebPlatformPlayer = ({ manifest, inputs }: Props) => {
     [accounts, manifest, navigation, device],
   );
 
+  const signMessage = useCallback(
+    ({ accountId, message }: { accountId: string, message: string }) => {
+      let formattedMessage: MessageData | null;
+
+      try {
+        const account = accounts.find(account => account.id === accountId);
+        invariant(account, "account not found");
+        formattedMessage = prepareMessageToSign(account, message);
+      } catch (error) {
+        return Promise.reject(error);
```

Comment:
I think we're just missing the tracking for success and failure

Reply:
@Justkant I don't find the info to put to the tracking for this US. Do you what info should be added?

Reply:
Just added the method by following the same implement as the other tracking methods.